spectral_cube/np_compat.py

```diff
@@ -14,7 +13,7 @@ def f(data, axis=None, keepdims=None):
             result = function(data, axis=axis)
         else:
             result = function(data, axis=axis, keepdims=keepdims)
-        if LooseVersion(np.__version__) >= LooseVersion('1.9.0') and hasattr(result, '__len__'):
+        if hasattr(result, '__len__'):
```

Comment:
Just checking on this change - are we sure this is OK?  Maybe we've already declared that spectral-cube requires >1.9 somewhere else at this point.... I don't recall

Reply:
numpy 1.9.0 was published 2014, and I assumed that it would not run under any of the more modern Python versions anymore. However, I didn't test.